backend/model/model.go

```diff
@@ -356,11 +357,13 @@ type SetupEvent struct {
 	ProjectID int                  `gorm:"uniqueIndex:idx_project_id_type"`
 	Type      MarkBackendSetupType `gorm:"uniqueIndex:idx_project_id_type"`
 }
+
 type ProjectFilterSettings struct {
 	Model
-	Project                    *Project
-	ProjectID                  int
-	FilterSessionsWithoutError bool `gorm:"default:false"`
+	Project                           *Project
+	ProjectID                         int
+	FilterSessionsWithoutError        bool `gorm:"default:false"`
+	AutoResolveStaleErrorsDayInterval int  `gorm:"default:0"`
```

Comment:
could this column be a `time.Duration` and stored as an `interval` in postgres? that way this can support finer time ranges than a day

Reply:
I considered that but when @julian-highlight and I were scoping this out, we came to the conclusion that setting below a day is not a good user experience. I'm going to keep it as an `int` since it makes the code more readable.

Reply:
Yea, I don't see a single use case to auto resolve errors < 1 day.

Reply:
that's fair, having an days in does keep the code simpler